meilisearch-types/src/document_formats.rs

```diff
@@ -19,15 +19,15 @@ type Result<T> = std::result::Result<T, DocumentFormatError>;
 pub enum PayloadType {
     Ndjson,
     Json,
-    Csv,
+    Csv(u8),
```

Comment:
Can we change this to be a struct with a named field i.e. `delimiter`, please? That would be better.